CMakeLists.txt

```diff
@@ -139,6 +140,19 @@ file(GLOB KVROCKS_SRCS src/*.cc)
 list(FILTER KVROCKS_SRCS EXCLUDE REGEX src/main.cc)
 
 add_library(kvrocks_objs OBJECT ${KVROCKS_SRCS})
+if(ENABLE_IPO)
+    include(CheckIPOSupported)
+    check_ipo_supported(RESULT result OUTPUT output LANGUAGES CXX)
+    if(result)
+        message(STATUS "IPO / LTO enabled")
+        set_property(TARGET kvrocks_objs PROPERTY INTERPROCEDURAL_OPTIMIZATION TRUE)
+        if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+            SET(CMAKE_STATIC_LINKER_FLAGS ${CMAKE_STATIC_LINKER_FLAGS} "-fuse-ld=lld")
```

Comment:
```suggestion
            set_property(TARGET kvrocks_objs APPEND_STRING PROPERTY LINK_FLAGS " -fuse-ld=lld")
```

Reply:
`CMAKE_STATIC_LINKER_FLAGS` does not work because `kvrocks_objs` is used for executables, not static libraries.